### PR TITLE
[JP] Update atom section in basic lesson

### PR DESCRIPTION
**Pull request by hinamiyagk**
Apply #470 changes to Japanese translation

### CODE REVIEW

**Comment by hinamiyagk** [comment]
#470

**Comment by hinamiyagk** [comment]
#480

**Comment by doomspork** [comment]
Thank you @hinamiyagk!  @3100 could you review this change?

File: jp/lessons/basics/basics.md [hunk 1]
```diff
@@ -114,6 +114,20 @@ iex> :true === true
 true
 ```
 
+Elixirのモジュールの名前もまたアトムです。`MyApp.MyModule`は、そのようなモジュールが宣言されていなくても有効ななアトムです。
```
Comment (3100) [inline jp/lessons/basics/basics.md]: Maybe a typo : '有効ななアトム'

Comment (hinamiyagk) [inline jp/lessons/basics/basics.md]: Fixed.

**Comment by 3100** [comment]
@hinamiyagk Thank you!

**Pull request merged**